_posts/2020/11/2020-11-06-configure-artipie-maven-repo.md

```diff
@@ -0,0 +1,122 @@
+
+```yml
+meta:
+  storage:
+    type: fs
+    path: /var/artipie/configs
+```
+
+If you want to use authentication for users then you need to add a section with credentials to the `artipie.yaml`. In this case, the file with settings should look like this:
+
+```yml
+meta:
+  storage:
+    type: fs
+    path: /var/artipie/configs
+  credentials:
+    type: file
+    path: _credentials.yml
+```
+
+After that you can install and run Artipie with docker:
+
+```bash
+$ docker run -d --name artipie \
+             -v $(pwd)/artipie.yaml:/etc/artipie.yml \
+             -v $(pwd):/var/artipie \
+             -p 8080:80 \
+             artipie/artipie:latest
+```
+
+## Configuring Artipie as a maven repository
+
+In the previous step we created a file with settings for Artipie. Now let's see what the configuration file for the repository should look like. For this, we need to create `configs` directory and the following `my-maven.yaml` file inside:
+
+```yml
+repo:
+  type: maven
+  storage:
+    type: fs
+    path: /var/artipie/data
+```
+
+If you added a section with credentials to the `artipie.yaml`, you also need to create a `_credentials.yml` file in the `configs` directory. Each node is a user name. The supported types of password can be found [here](https://github.com/artipie/artipie/blob/700eb89352126e6f1bd12a0d3ff668abf2b44048/README.md#multitenancy) The file with credentials should look like this:
+
+```yml
+credentials:
+  alice:
+    type: plain
+    pass: 123
+  bob:
+    type: plain
+    pass: qwerty
+```
+
+As a results, you will have the following files structure:
+
+```bash
+.
+├── artipie.yaml
+├── configs
+    ├── my-maven.yaml
+    └── _credentials.yml # optinal
+```
+
+If you have completed all the instructions correctly now you should be able to use the repository: deploy and install Maven artifacts.
+
+## Configuring project to use your Artipie maven repository
+
+At this step, I will show you how to configure your project to use created Artipie maven repository. If you want to `mvn deploy` your project to Artipie, add the following to project's `pom.xml`:
+
+```xml
+<distributionManagement>
+    <repository>
+        <id>artipie</id>
+        <url>http://localhost:8080/my-maven</url>
+    </repository>
+</distributionManagement>
+```
+
+In case you want to `mvn install` deployed packages from Artipie then add the following to project's `pom.xml`:
+
+```xml
+<repositories>
+    <repository>
+        <id>artipie</id>
+        <url>http://localhost:8080/my-maven</url>
+    </repository>
+</repositories>
+```
+
+## Advanced options
+
+Apart from demonstrated possibilities, Artipie Maven supports: group repositories, proxies and user groups.Check out the corresponding
+[documentation section](https://github.com/artipie/artipie/tree/master/examples/maven)
+for more details.
+
+## Conclusion
+
+In this blog post I explained how to set up own Maven repository with Artipie and how to
+configure a Maven project to use it. 
```

Comment:
@genryxy this section again feels too formal, like something from an scientific article, I'd suggest to make it more informal. Maybe it feels that way because of labeled sections, blog post are expected to be less formal and be more like a story, not a document